qemu/tests/snp_basic_config.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from avocado.utils import process
+from virttest import data_dir as virttest_data_dir
+from virttest import env_process, error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu snp basic test on Milan and above host:
+    1. Check host snp capability
+    2. Boot snp VM
+    3. Verify snp enabled in guest
+    4. Check snp qmp cmd and policy
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    error_context.context("Start sev-snp test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
+    snp_module_path = params["snp_module_path"]
+    if os.path.exists(snp_module_path):
+        f = open(snp_module_path, "r")
+        output = f.read().strip()
+        f.close()
+        if output not in params.objects("module_status"):
+            test.cancel("Host sev-snp support check fail.")
+    else:
+        test.cancel("Host sev-snp support check fail.")
+    if params.get("singlesocket_host_check") == "yes":
+        s, o = process.getstatusoutput(params.get("singlesocket_check_cmd"), shell=True)
```

Comment:
remove singlesocket_host_check param in cfg, just check if singlesocket_check_cmd is defined,e.g.
singlesocket_check_cmd = params.get("singlesocket_check_cmd")
if singlesocket_check_cmd:

Reply:
you may use process.system to return the status only if you dont need the output

Reply:
Updated and checked, this way works for the case.